### ISSUE TITLE
Feature Suggestion: Comment Mode separate from Edit Mode

### ISSUE THREAD

**Issue by mkozdon** [SabakiHQ/Sabaki#966]
As user, I'd like to have an option to edit comments while creating a game tree. 
Currently, when I am creating an SGF with multiple variations I need to switch back and forth between game/edit mode.

Separating Comment Mode from Edit Mode would enable creating variations with comments without extra clicking.

As alternative solution, "make move" could be added to Edit toolbar. In a manner, that moves can be added while being in Edit Mode instead of just adjusting the stones on current move.

**Comment by Chewt** [comment]
An alternative option would be a slight change in workflow. The ctrl+E shortcut toggles edit mode and automatically focuses the comment text box, so to quickly add moves and edit comments you could adopt the following flow:

1. Make a move
2. Press CTRL+E and type your comment
3. Press CTRL+E again and make another move
4. Repeat steps 1-3

Utilizing shortcuts would be a much simpler solution than changing the code, and learning other shortcuts will make you even faster at using Sabaki!

**Comment by wilczasty** [comment]
I think you've misinterpreted my intention :) I don't want to solve my problem, I want to help make this app more user friendly.
I know the shortcuts and I understand, that workaround is better (for dev) than changing the behavior.

Still these are at least 2 additional steps per comment without any value added for the process (end user).
While reviewing the game multiple comments and branches are created and it becomes a visible inconvenience.
In other words - why user needs to press CTRL+E 30, 50, 100 times in order to perform a simple task of adding comments? 
Other SGF editors have solved the problem just by not creating it - comment section is always active. That is why I'm raising this as a feature/change request for consideration.
